mmc-google-analytics/src/main/java/info/novatec/metricscollector/google/ResponseParser.java

```diff
@@ -19,19 +20,24 @@
 
     public void parse(GetReportsResponse reportsResponse, List<Metrics> targetListOfMetrics) {
         Report report = reportsResponse.getReports().get(0);
+        List<ReportRow> rows = report.getData().getRows();
         List<String> dimensionsHeaders = report.getColumnHeader().getDimensions();
         List<MetricHeaderEntry> metricsHeaders = report.getColumnHeader().getMetricHeader().getMetricHeaderEntries();
         final int[] numberOfMCollectedMetrics = { 0 };
```

Comment:
is "numberOfMCollectedMetrics" correct or should it be "numberOfCollectedMetrics" ?